libafl/src/bolts/os/windows_exceptions.rs

```diff
@@ -315,8 +318,12 @@ unsafe fn internal_handle_exception(
     }
 }
 
-/// Internal function that is being called whenever an exception arrives (stdcall).
-unsafe extern "system" fn handle_exception(exception_pointers: *mut EXCEPTION_POINTERS) -> c_long {
+/// Function that is being called whenever an exception arrives (stdcall).
+/// # Safety
+/// This function is unsafe because it is called by the OS
```

Comment:
Maybe something like `Should never be called directly, only called by the OS` or something?

Reply:
Any other better wording for this?

Reply:
Just somehow indicate that's it's never safe if you call it manually (or is it?)

Reply:
Is there any reason to call this for users? Else just weite it's not safe to call directly

Reply:
We found at least one reason to call it from our code, that's in UnhandledExceptionFilter hook. If this case is not worh mentiong in comments, we can write `Should never be called directly, only called by the OS` -- would that be ok?